rpc/openapi/openapi.yaml

```diff
@@ -1002,6 +1002,50 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /prove_shares:
+    get:
+      summary: Prove shares for a given share range
+      description: |
+        Generates a shares to data root inclusion proof
+        for a set of shares referenced by their range and
+        the block height they belong to.
+
+        The share range is end exclusive.
+      operationId: prove_shares
+      tags:
+        - Info
+      parameters:
+        - in: query
+          name: height
+          description: The block height
+          schema:
+            type: integer
+            default: 1
+            example: 1
+        - in: query
+          name: startShare
+          description: The starting share index
+          schema:
+            type: integer
+            default: 1
+            example: 1
+        - in: query
+          name: endShare
+          description: The ending share index
```

Comment:
should this say something about how the endShare is exclusive?

Reply:
```suggestion
          description: The end exclusive ending share index.
```

@rootulp like this?

Reply:
Sure that works